### PR TITLE
fix: use certification hours consistently

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Since the value appearing on the /learn map was hardcoded, it could have
gone out of sync with the certifications, which were relying on
certification-settings

Also, for simplicity (and because it's hard to estimate accurately), all
the certifications are set to 300 hours.  Fullstack being the exception,
as it is a set of certifications.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by ojeytonwilliams** [comment]
After a discussion in chat, the consensus was that the 300 hours on the learn map can go.  The certifications don't need changing.  Not beyond the changes already present in this PR.

**Review by raisedadead — DISMISSED** [review]

**Comment by raisedadead** [comment]
So - the PR is unblocked now because we have a consensus?

**Comment by ojeytonwilliams** [comment]
It could go in, yeah, but I was going to remove the 300hrs from the learn map.

**Comment by ahmadabdolsaheb** [comment]
I vote to remove it from map. 
Here is my reasoning from chat for our reference:
"We do communicate the difficulty of certificates when people sign in for freecodecamp, so there is no need for those 300hr estimates."

**Comment by ojeytonwilliams** [comment]
@ahmadabdolsaheb do you think we should remove the "Thousands of hours of challenges" bit from the map and mention that in the intro?

**Comment by ahmadabdolsaheb** [comment]
Yes. :)

**Comment by ojeytonwilliams** [comment]
After removing the hours:

image
image

File: config/certification-settings.ts [hunk 1]
```diff
@@ -54,9 +54,9 @@ export const certIds = {
 };
 
 export const completionHours = {
-  [certTypes.frontEnd]: 400,
-  [certTypes.backEnd]: 400,
-  [certTypes.dataVis]: 400,
+  [certTypes.frontEnd]: 300,
+  [certTypes.backEnd]: 300,
+  [certTypes.dataVis]: 300,
```
Comment (ahmadabdolsaheb) [inline config/certification-settings.ts:59]: So the things with these is that, long time ago we had only 3 certs. And presumably would take users longer to complete each certification.

**Review by ahmaxed — APPROVED** [review]

File: config/certification-settings.ts (hunk 1 above)
Comment (ahmadabdolsaheb) [reply config/certification-settings.ts:59]: Although we are sunsetting that part of freeCodeCamp, there might be people that find issue with this change.

I think we move forward with changing everything to 300 and revert if an issue is raised.

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**